### ISSUE TITLE
drop the need to `extend_timeout`

### ISSUE THREAD

**Issue by emphasize** [OpenVoiceOS/ovos-ocp-audio-plugin#62]
This is a common message that is consumed by `handle_skill_response` during an active search:

```
{'phrase': 'regenbogen', ... , 'results': [{}], 'searching': False}
``` 
_(`searching`is always False no matter if a skill is searching)_

to extend the timeout you need to `self.extend_timeout()` in a CPSkill which is sending

```
{'phrase': 'regenbogen', ..., 'timeout': x , 'searching': True}
```

to trigger the respective cases
https://github.com/OpenVoiceOS/ovos-ocp-audio-plugin/blob/fced5de8d2c3f2803a88f54578db63f986082356/ovos_plugin_common_play/ocp/search.py#L157-L170
 

While i see the potential goal to give the skill the freedom to extend to their liking (btw no skill is actually doing that - resulting in a problem when using `OCPQuery().wait()`), i don't like that implementation. It feels clumsy sending _a search_ with `"searching": False` just to need another message to send the timeout amount with `"searching": True` cluttering the bus in the process.

**Comment by JarbasAl** [comment]
I agree this is unclean, but it was the way original mycroft common play did (where OCP was born from)

I'm in favor of refactoring this one, but I'd like to keep the extend_timeout functionality around, OCP settings allows users to disable or limit this extension to protect against bad skills

**Comment by emphasize** [comment]
Then we have to _ensure_ CPSkills are actually doing this. I'm running into problems with only 3(!) skills around (tunein, youtube, news). And if more and more News stations are dripping in, this only gets worse. Tunein is naturally sending only a handful.

Besides, those have also to cap the `match_confidence` worth sending.

**Comment by emphasize** [comment]
I'm hawking on this. what if in OVOSCommonPlaybackSkill Class ([#](https://github.com/OpenVoiceOS/OVOS-workshop/blob/693a213a666bdc17e1c82809d4eff838a5e6f732/ovos_workshop/skills/common_play.py#L264)) we

```python
        for handler in self._search_handlers:
            ...
            
            # handler might return a generator or a list
            if isinstance(results, list):
                # inject skill id in individual results, will be needed later
                # for proper playback handling
                for idx, r in enumerate(results):
                    results[idx]["skill_id"] = self.skill_id
                self.bus.emit(message.response({"phrase": search_phrase,
                                                "skill_id": self.skill_id,
                                                "skill_name": self.name,
                                                "thumbnail": self.skill_icon,
                                                "results": results,
                                                "timeout": self.search_timeout}))
```

making `self.search_timeout` a property? We can even make it a user setting.

**Comment by emphasize** [comment]
you mean instead of a separate message that carries the amount and searching true. Where`s the problem?
